src/js/playlist/track.ts

```diff
@@ -1,3 +1,12 @@
+type PlaylistItemTrack = {
+    file?: string;
+    kind: 'captions' | 'metadata' | 'thumbnails' | 'chapters';
```

Comment:
We might want to enforce the value for `kind` and boolean type for `default` since we could end up with anything thanks to `Object.assign`.

Reply:
Not sure exactly what you mean by `enforce`?

Reply:
> enforce?
After `Object.assign` replace invalid values with their defaults.

Reply:
ah, updating